README.md

```diff
@@ -89,7 +91,7 @@ npm install netlify-cli -g
 netlify deploy # enter ./build as the deploy path
```

Comment:
@goodroot That seems like a decent idea. At what point did you encounter that error?

Reply:
@goodroot At what point did you encounter that error?

Reply:
If one goes right to the repo and tries the deploy steps ("Oh, I can deploy, I want to see it running first").  The error appears once the netlify deploy is complete and 'staged' -- it works, but returns that error from the running reference ui.

Also, it might be helpful to start the deploy steps with an `npm install`, as the build will fail if packages are not installed.